Controllers/AdminController.cs

```diff
@@ -33,16 +33,10 @@ public Sighting ConfirmSighting([FromRoute] int id)
         }
 
         [HttpPost("updateAndConfirmSighting/{id}")]
-        public IActionResult updateAndConfirmSighting([FromBody] Sighting SightingToUpdate)
+        public Sighting updateAndConfirmSighting([FromBody] Sighting SightingToUpdate)
         {
-            if (!ModelState.IsValid)
```

Comment:
how come we removed this? Is there not some validation on sighting that might be useful?